planter/README.md

```diff
@@ -0,0 +1,16 @@
+# Planter
+
+Planter is a container + wrapper script for your bazel builds.
+It will run a docker container as the current user that can run bazel builds
+in your `$PWD`. It has been tested on macOS and Linux against 
+`kubernetes/test-infra` and `kubernetes/kubernetes`.
```

Comment:
This is true now, but will it be true in a week? Will we be continuously testing this?

Reply:
I will be testing this periodically and keeping up with the bazel build changes, it should not break unless something is wrong with bazel. The only things that needed special treatment are because:

a) Docker as user doesn't set up the user in a very linux like fashion so tools break on looking up the user.
b) Test-Infra has a leaky bazel setup that doesn't manage deps properly.
c) We do need a wrapper script to mount in the appropriate directories (this definitely should **not** change).

If this turns out to not be used / I quit maintaining it (hopefully people do just use native bazel!) we can mark it as deprecated.

Reply:
Or more concisely, if it becomes not-true, someone should open an issue and I will investigate. Supporting mac and linux for our repos is the goal for now.